diff --git a/dist/plugins/next.d.ts b/dist/plugins/next.d.ts
index 0ae554ab8855bd02482d3acd33d4a0173620d1ac..2976671117e4d0ab114d66e309098893b73480b8 100644
--- a/dist/plugins/next.d.ts
+++ b/dist/plugins/next.d.ts
@@ -5,4 +5,4 @@ interface VercelToolbarConfig {
 }
 declare function withVercelToolbar(config?: VercelToolbarConfig): (nextConfig: NextConfig) => NextConfig;
 
-export { VercelToolbarConfig, withVercelToolbar };
+export { VercelToolbarConfig, withVercelToolbar as default };